internal/grid/trace.go

```diff
@@ -129,7 +129,11 @@ func (c *muxClient) traceRoundtrip(ctx context.Context, t *tracer, h HandlerID,
 		trace.HTTP.ReqInfo.Path = trace.Path
 	} else if v != nil {
 		// Print exported fields as single request to path.
-		trace.Path = fmt.Sprintf("%s?req=%s", trace.Path, url.QueryEscape(fmt.Sprintf("%+v", v)))
+		obj := fmt.Sprintf("%+v", v)
+		if len(obj) > 1024 {
```

Comment:
```suggestion
		if len(obj) > 1024 {
```

still 1k is a lot? @klauspost

Reply:
It is only internal traces... So I think it is safe to say that it is rarely enabled.... What would you like it to be?

Reply:
I am thinking that can this be just like 64bytes and not more than that. Seeing entire fileinfo is not a requirement really TBH.